about.md

```diff
@@ -100,7 +100,6 @@ Operating systems that already ship Calamares:
 - [Feren OS](https://ferenos.weebly.com/){:target="_blank"}
 - [Garuda Linux](https://garudalinux.org/){:target="_blank"}
 - [GeckoLinux](http://geckolinux.github.io/){:target="_blank"}
-- [Kannolo](https://kannolo.sourceforge.io/){:target="_blank"} (Fedora Remix)
```

Comment:
website is still active, dev is still active in calamares, so won't be removed from list, unless the developer wants it removed

Reply:
You should remove BBQLinux as the website is no longer active. (Domain has been taken over)